crates/bevy_ecs/src/schedule/system_descriptor.rs

```diff
@@ -126,10 +129,10 @@ pub trait ParallelSystemDescriptorCoercion<Params> {
     fn label(self, label: impl SystemLabel) -> ParallelSystemDescriptor;
 
     /// Specifies that the system should run before systems with the given label.
-    fn before(self, label: impl SystemLabel) -> ParallelSystemDescriptor;
+    fn before<Marker>(self, label: impl AsSystemLabel<Marker>) -> ParallelSystemDescriptor;
```

Comment:
Is `Marker` what we call `Params` (iirc) elsewhere? I.e. the type parameter for disambiguation.

Edit: Ah, apparently both `Params` and `Marker` already exist, because rustc makes us suffer